src/pages/extending-agent-app/index.mdx

```diff
@@ -8,7 +8,7 @@ desc: "You can extend the LiveChat Agent App with widgets, buttons, and other UI
 
 # Introduction
 
-This document presents how you can change and extend the LiveChat App and its interface, so it is personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
+This document presents how you can change and extend the LiveChat and HelpDesk Apps and their interfaces, so they're personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
```

Comment:
Noticed that this paragraph was missed by me when adjusting docs to also talk about HD